tests/qam-kgraft/reboot_restore.pm

```diff
@@ -59,8 +59,8 @@ sub run() {
     script_run("cat /tmp/lsboot");
     save_screenshot;
 
-    script_run("basename /boot/initrd-\$(uname -r) | sed s_initrd-__g | sed s_-default__g | tee > /dev/$serialdev", 0);
-    my ($kver) = wait_serial(qr/^\d\.\d+\.\d+-\d+\.\d+/) =~ /(^\d\.\d+\.\d+-\d+\.\d+)\s/;
+    script_run("basename /boot/initrd-\$(uname -r) | sed s_initrd-__g | sed s_-default__g > /dev/$serialdev", 0);
```

Comment:
here is important get result to serial console for use in next part of  script, not for print it to screen